### PR TITLE
[jythonscripting] updated readme

### PR DESCRIPTION
**Pull request by HolgerHees**
changed script and default library path

### CODE REVIEW

**Comment by HolgerHees** [comment]
@wborn could you please review this tiny pull request? Currently I have no permission to request a review. That's why I mention you directly.

**Review by wborn — APPROVED** [review]
Thanks!

**Comment by wborn** [comment]
Maybe @kaikreuzer can add you to the Contributors team? Then you can also request reviews and assign labels yourself.

**Comment by jlaur** [comment]
> Maybe @kaikreuzer can add you to the Contributors team? Then you can also request reviews and assign labels yourself.

You beat me. 😄

image

**Comment by HolgerHees** [comment]
@kaikreuzer when a good reputation is required... 10 years ago I was the original author of comfoair binding at openhab 1.0. :-)

**Comment by kaikreuzer** [comment]
@HolgerHees That is very honorable of you - I would add you twice to the contributor team, if I could. 😉 
tl;dr: Done!
